time stamp
uint8[] id_list

uint16[] model_number
string[] drive_mode
string[] operation_mode
uint32[] homing_offset